lib/pacemaker/pcmk_sched_bundle.c

```diff
@@ -892,11 +892,16 @@ output_replica_actions(pe__bundle_replica_t *replica, void *user_data)
     return true;
 }
 
+/*!
+ * \internal
+ * \brief Output a summary of scheduled actions for a bundle resource
+ *
+ * \param[in,out] rsc  Bundle resource to output actions for
```

Comment:
I was very surprised to see that an output function has side effects, but sure enough, in `pcmk__output_resource_actions()`.

Reply:
Every output function has side effects on the output object :)